core/translations/i18n/en.json

```diff
@@ -49,82 +49,7 @@
   "Unloading module \"${0}\".": "Unloading module \"${0}\".",
   "What kind of shit platform is this that doesn't even support sending messages?": "What kind of shit platform is this that doesn't even support sending messages?",
   "Working on it...": "Working on it...",
-
```

Comment:
> "Skipping \"${0}\". It isn't a Kassy module.": "Skipping \"${0}\". It isn't a Kassy module."

An instance of **Kassy** snuck in past the PR, across the other translation as well.